projects/cincinnati/cincinnati.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: v1
+kind: List
+items:
+- apiVersion: v1
+  kind: BuildConfig
+  metadata:
+    name: cincinnati-build-root-latest
+    labels:
+      ci.openshift.io/source-repo: cincinnati
+  spec:
+    successfulBuildsHistoryLimit: 1
+    failedBuildsHistoryLimit: 2
+    output:
+      to:
+        kind: ImageStreamTag
+        name: cincinnati-build-root:latest
+    strategy:
```

Comment:
You need to specify the GitSource here

Reply:
I'm not sure if I understand this right. please take another look